### ISSUE TITLE
Fails to open files in neovim 0.2

### ISSUE THREAD

**Issue by lohfu** [francoiscabrol/ranger.vim#24]
After upgrading to Neovim 0.2 it appears this plugin has broken.

After either exiting ranger or trying to open a file with it exits with an error.

```
E118: Too many arguments for function: 16
```

The 16 increases by 1 on subsequent calls.

I am running Arch Linux, Neovim 0.2.0-1 from the official repos and ranger:

```
ranger-stable 1.8.1

Python 3.6.1 (default, Mar 27 2017, 00:27:06) 
[GCC 6.3.1 20170306]
```

**Comment by francoiscabrol** [comment]
I have the same version of ranger (1.8.1) and neovim 0.2 works well for me... 
Does someone else can reproduce it? 
Do you have an idea about what does it mean?

**Comment by lohfu** [comment]
My my, i am terribly sorry but it appeared i had actually implemented my own version of your plugin... was sure i was using this. Testing your plugin again it seems to work, but RangerWorkingDirectory is not opening ranger at all.

The reason i wasn't using your plugin was i wanted to open ranger with the current file selected when not opening the working directory. I got both the RangerWorkingDirectory command working and selecting the current file... Would you be interested in a PR or do you not want to select the current file when opening ranger?

**Comment by francoiscabrol** [comment]
Humm I am not sure to understand... what you want is selecting the current file when opening the current file's folder with RangerCurrentDirectory?

**Comment by francoiscabrol** [comment]
Because RangerWorkingDirectory open the workspace directory from where vim was opened... The current file can't be selected (except if the file is in this folder)

**Comment by lohfu** [comment]
> Humm I am not sure to understand... what you want is selecting the current file when opening the current file's folder with RangerCurrentDirectory?

Yes exactly... I do this simply by using the `--selectfile` option when opening ranger...

**Comment by rosshadden** [comment]
@lohfu I had the same error (and the same situation where I'm using [my own custom version of this code, which handles selecting files](https://github.com/rosshadden/dotfiles/blob/master/src/.config/nvim/config/modules/files.vim#L25-L67)).  I fixed it simply by [adding an extra argument to `on_exit`](https://github.com/rosshadden/dotfiles/commit/f12801ff4b6387317c8903e68099dbae03d627c1#diff-1b40283d06900d402b4ea2e0460b880e).

**Comment by francoiscabrol** [comment]
I think the bug when you want to run RangerWorkingDirectory is fixed with #26

**Comment by francoiscabrol** [comment]
about the behaviour for opening the selected file, I will add it
